src/azure/storage/signer.rs

```diff
@@ -127,8 +136,13 @@ impl Signer {
     }
 
     /// Signing request with query.
-    pub fn sign_query(&self, req: &mut impl SignableRequest, cred: &Credential) -> Result<()> {
-        let ctx = self.build(req, SigningMethod::Query(Duration::from_secs(1)), cred)?;
+    pub fn sign_query(
+        &self,
+        req: &mut impl SignableRequest,
+        expire: Duration,
+        cred: &Credential,
+    ) -> Result<()> {
+        let ctx = self.build(req, SigningMethod::Query(expire), cred)?;
```

Comment:
@Xuanwo Please note that we have modified the signature of this function, and we need to pay attention to compatibility when releasing the version.

Reply:
Thanks! We should release v0.10 instead!